public/js/login.js

```diff
@@ -1,4 +1,6 @@
 /* Process a user login request. */
+var failedLogin = '<div class="chip white-text red darken-4">Invalid username or password<i class="close material-icons">close</i></div>';
```

Comment:
The changes in this file are the same as in PR33. Is that by design? Shouldn't they be different changes in case the other PR is rejected?

Reply:
idk what is this mess. Yeah I will removed all of those now.